app/templates/rentals.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <h2>Welcome!</h2>
+  <p>
+    We hope you find exactly what you're looking for in a place to stay.
```

Comment:
We should do 1 of 2 things for this text:
1. Make the text more generic since it shows on both show and list
2. Make the text dynamic, to show different instructions based on the route.